docs/manual/resource-leak-checker.tex

```diff
@@ -588,6 +596,240 @@
 will only use \<@MustCallUnknown> as a default when encountering a language
 feature that it is unable to reason about.
 
+\sectionAndLabel{Arrays of resources}{resource-leak-arrays}
+Arrays of resources are a particular challenge for static analysis, due to the inherently dynamic nature that arrays are accessed with.
```

Comment:
"are accessed with" is clumsy in English. I suggest "due to the inherently dynamic nature of array accesses" instead.